app/schemas/models/user.coco.coffee

```diff
@@ -461,6 +461,12 @@ _.extend UserSchema.properties,
   library: c.object {}, {
     profileId: { type: 'string' }
   }
+  related: c.array(
+    { description: 'related accounts to this user' },
+    c.objectId(
+      { description: 'userId of the account currentUser is related to' }
+    )
+  )
```

Comment:
Do we want this to be just the related user `_id`s, or do we want to denormalize some properties here (like `role` and `name`) so that we don't have to do a fetch in order to display the account switcher info in the top right?

(I guess it might be easier to just fetch data in order to show that list of related users there than to keep the denormalized data reasonably up-to-date.)

Reply:
Ya, de-normalizing will have issues with data plus we might need to keep adding data from users to here as and when new features are needed plus backfiling so it's better to only store related account things here. 

I have made this an array of objects instead of an array of object-ids though now since I feel we will need to store some information where the account that is added is verified or not, if we want to send an email for verification then we might need to store that too.